components/menu/test/menu-item-mixin.test.js

```diff
@@ -0,0 +1,62 @@
+import { defineCE, expect, fixture, oneEvent } from '@open-wc/testing';
+import { LitElement } from 'lit-element/lit-element.js';
+import { MenuItemMixin } from '../menu-item-mixin.js';
+
+const tag = defineCE(
+	class extends MenuItemMixin(LitElement) {}
```

Comment:
Was able to kill all of the test elements and replace them with simple classes like this. 🚀